hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/disk/OnDiskKeySerializer.java

```diff
@@ -72,7 +72,7 @@ public OnDiskKeySerializer(@NonNull final StateMetadata<K, ?> md) {
     }
 
     @Override
-    public int getSerializedSize() {
+    public int getSerializedSize(long var1) {
```

Comment:
will be helpful to have a different name than `var1`. same in other `XXSerializer` classes

Reply:
Updated it with long dataVersion as you suggested. Thanks.